itests/org.jboss.tools.openshift.ui.bot.test/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jboss.tools.openshift</groupId>
+		<artifactId>itests</artifactId>
+		<version>3.4.1-SNAPSHOT</version>
+	</parent>
+	<groupId>org.jboss.tools.openshift.tests</groupId>
+	<artifactId>org.jboss.tools.openshift.ui.bot.test</artifactId>
+	<packaging>eclipse-test-plugin</packaging>
+
+	<properties>
+		<systemProperties>
+			-Dtest.configurations.dir=${configurations.dir} -Dopenshift.server=${openshift.server} -Dopenshift.xserver=${openshift.xserver} -Dopenshift.username=${openshift.username} -Dopenshift.password=${openshift.password} -Dopenshift.token=${openshift.token} -Dgithub.username=${github.username} -Dgithub.password=${github.password} -Dopenshift.authmethod=${openshift.authmethod} -Dsecurestorage.password=${securestorage.password} -Dopenshift.nexus.mirror=${openshift.nexus.mirror}
+		</systemProperties>
+		<test.class>org.jboss.tools.openshift.ui.bot.test.OpenShift3SmokeBotTests</test.class>
+		<surefire.timeout>10800</surefire.timeout>
+		<openshift.server></openshift.server>
+		<openshift.xserver></openshift.xserver>
+		<openshift.username></openshift.username>
+		<openshift.password></openshift.password>
+		<openshift.token></openshift.token>
+		<openshift.authmethod></openshift.authmethod>
+		<openshift.nexus.mirror></openshift.nexus.mirror>
+		<github.username></github.username>
+		<github.password></github.password>
+		<securestorage.password></securestorage.password>
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<configuration>
+					<dependency-resolution>
+						<optionalDependencies>ignore</optionalDependencies>
+					</dependency-resolution>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-compiler-plugin</artifactId>
+				<configuration>
+					<optionalDependencies>ignore</optionalDependencies>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<configuration>
+					<appArgLine>-pluginCustomization ${basedir}/pluginCustomization.ini</appArgLine>
```

Comment:
use ${requirementsDirectory} instead of ${basedir}. ITests profile from parent pom downloads pluginCustomization.ini into requirementsDirectory.